src/Trixi.jl

```diff
@@ -161,7 +163,8 @@ export boundary_condition_do_nothing,
        BoundaryConditionNeumann,
        boundary_condition_noslip_wall,
        boundary_condition_slip_wall,
-       boundary_condition_wall
+       boundary_condition_wall,
```

Comment:
Can we delete this? Or is `boundary_condition_wall` being used somewhere?

Reply:
The acoustic perturbation equations use this boundary condition in `examples/tree_2d_dgsem/elixir_acoustis_gauss_wall.jl`